Gemfile

```diff
@@ -38,7 +38,8 @@ gem "dnsruby", "~> 1.60.0", require: false
 gem "email_validator", "~> 1.6"
 
 # HTTP library wrapper
-gem "faraday", "~> 0.9.2", require: false
+gem "faraday", "~> 0.9.2"
+gem 'faraday_middleware', '~> 0.13.1'
```

Comment:
Do we need this?

Reply:
We do need this to send JSON to eyeshade. I don't think we had any `POST` requests to eyeshade before but when I did not include this the request was unable to be sent properly.

Reply:
https://github.com/brave-intl/publishers/pull/1660/commits/8500d5d4ffa56959ddd1d4705a6c6eb481f2dd7c